tests/Doctrine/Tests/ORM/Hydration/ObjectHydratorTest.php

```diff
@@ -200,7 +199,7 @@ public function testSimpleMultipleRootEntityQuery(): void
     }
 
     /**
-     * SELECT PARTIAL u.{id, name} AS user, PARTIAL a.{id, topic}
+     * SELECT u AS user, a
      *   FROM Doctrine\Tests\Models\CMS\CmsUser u, Doctrine\Tests\Models\CMS\CmsArticle a
      */
     public function testSimpleMultipleRootEntityQueryWithAliasedUserEntity(): void
```

Comment:
I don't think these tests are _about_ partial objects. For instance, according to the title, this one seems to be about testing that the objecthydrator works when the rsm as several root entities, and one of them is aliased. I think they still make sense, even after removing partial objects.

Reply:
Seems like the partial objects really were just unrelated to the tests.